specs/RequestAnimationFrame/Overview.html

```diff
@@ -2,18 +2,18 @@
 <html>
 
 <head>
-  <title>Timing control for script-based animations</title>
+  <title>Animation Frames</title>
   <meta charset='utf-8'>
-  <meta http-equiv="Refresh" content="5; URL=http://w3c.github.io/animation-timing/">
+  <meta http-equiv="Refresh" content="5; URL=https://html.spec.whatwg.org/#animation-frames">
```

Comment:
Nit: Redirecting to the multipage variant may be friendlier to users

Reply:
Done. Thank you for the suggestions!